### PR TITLE
Added final step method in LoginPageStepDef + created dashboard_locat…

### PR DESCRIPTION
**Pull request by k-brennan**
…or.yml in prep for next steps after logging in

### CODE REVIEW

File: conf/dashboard_locator.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    # Add Home Page ID locators below:
+    
+
+  name:
+    # Add Home Page NAME locators below: 
+
+
+  css:
+    # Add Home Page CSS locators below:
+    dashboard.myProducts.header: '.color333.dbl_tableTitle.pull-left'
```
Comment (davidansell) [inline conf/dashboard_locator.yml]: I think I would probably user this ID am my locator: products-sec

File: src/test/java/com/nttdata/stepdefs/LoginPageStepDefs.java [hunk 2]
```diff
@@ -34,6 +34,8 @@ public void i_should_be_asked_for_input_providing_account_login_details() {
 	public void i_login_using_valid_credentials() {
 		loginPage.userId.sendKeys("kieran.brennan@nttdata.com");
 		loginPage.userPasswd.sendKeys("!arsenaL14");
+		assertThat("Login with credentials has failed", loginPage.modalLoginButton.isPresent(), equalTo(true));
```
Comment (davidansell) [inline src/test/java/com/nttdata/stepdefs/LoginPageStepDefs.java]: Add a wait to this assert

File: conf/dashboard_locator.yml (hunk 1 above)
Comment (k-brennan) [reply conf/dashboard_locator.yml]: like this you mean?
products-sec: '.color333.dbl_tableTitle.pull-left'

File: src/test/java/com/nttdata/stepdefs/LoginPageStepDefs.java [hunk 3]
```diff
@@ -34,6 +34,8 @@ public void i_should_be_asked_for_input_providing_account_login_details() {
...
+		loginPage.modalLoginButton.waitUntil(displayed).click();
```
Comment (davidansell) [inline src/test/java/com/nttdata/stepdefs/LoginPageStepDefs.java]: Ok I would combine your last two statements with something like:
loginPage.modalLoginButton.waitUntil(enabled).click()

Enabled better that displayed.

**Pull request merged**